plugins/org.eclipse.n4js/src/org/eclipse/n4js/N4JSGlobals.java

```diff
@@ -75,10 +75,13 @@
 	 * Unmodifiable list containing {@link #N4JSD_FILE_EXTENSION}, {@link #N4JS_FILE_EXTENSION},
 	 * {@link #JS_FILE_EXTENSION}.
 	 */
+	// TODO what about N4MF ?
+	// Clarify usage and why N4MF is excluded
```

Comment:
Good point. But can this be resolved during this task (even if unrelated)?
We should avoid adding more `TODO`s to the code, as far as possible. (esp. if they don't have an issue ID attached)

Reply:
added ticket number. This is used in the builder, runtime core, headless builder....